package.json

```
@@ -48,7 +48,7 @@
     "xo": "0.36.1"
   },
   "engines": {
-    "node": ">=14.17"
+    "node": "^17 || ^16 || ^14.17"
```

```suggestion
    "node": ">=16 || ^14.17"
```

lets keep the upper bound open